src/wallet/types.ts

```diff
@@ -40,3 +40,8 @@ export type AddressKeys = {
 export type WalletData = { mnemonic: string; index: number };
 
 export type WalletNodes = { spending: Node; viewing: Node };
+
+export enum LeptonEvent {
+  Scanned = 'scanned',
+  Nullified = 'nullified',
+}
```

Comment:
Because these are events on specific classes' EventEmitters, we should rename for clarity: 

- WalletScanComplete
- ContractNullifierReceived